cdk/v2/utils.js

```diff
@@ -0,0 +1,81 @@
+const path = require("path");
+const fs = require("fs");
+const {
+  WorkflowExecutionError
+} = require("rudder-workflow-engine/build/errors");
```

Comment:
We should expose the modules or functions in a better way. Just a note @koladilip (even I missed this with prev version of CDK 🙂)

Reply:
I agree. It's a matter of publishing the package from the `build` directory. Let me make that change as part of the next release.

Reply:
We can just use the main field in package.json
https://stackoverflow.com/questions/22512992/how-to-use-the-main-parameter-in-package-json#22513200

Reply:
Yes. We already have defined it. However, not all the types (ex: `error.ts`) are exported from `index.ts`.
Hence, I could fix this issue for some of the imports.
We can resolve others here once we update the package in the next release.

Reply:
https://github.com/rudderlabs/rudder-workflow-engine/pull/14